packages/unmock-core/tslint.json

```diff
@@ -4,6 +4,8 @@
         "tslint:recommended"
     ],
     "jsRules": {},
-    "rules": {},
+    "rules": {
+        "arrow-parens": false
```

Comment:
The ts-lint of packages should probably be inherited from the root tslint.json if possible? Not a topic for this PR of course.

Reply:
Ah, I just saw it was present in the `unmock-node` linting, so I copied from there. It was annoying that one would highlight it as error and the formatter would remove the fix.

Reply:
Yup, maybe `tslint-config-prettier` would need to be setup for all packages as well to avoid the conflicts.